src/util/KnowledgeGraphUtils.js

```diff
@@ -327,6 +327,21 @@ export default class KnowledgeGraphUtils {
     return result;
   }
 
+  //eslint-disable-next-line
+  static stringToColor(str) {
```

Comment:
Could you please add a comment in here explaining what the algorithm is doing? Thanks!

Reply:
Added comments, let me know if you need me to explain it further.